### PR TITLE
Better fix for core/quotes tests that broke after xUnit

### PR DESCRIPTION
**Pull request by majocha**
We had some tests that depended on existence of artifacts from another test.
The problem was revealed because xUnit does not guarantee order of execution.

### CODE REVIEW

File: tests/fsharp/tests.fs [hunk 1]
```diff
@@ -864,6 +853,28 @@ module CoreTests =
             testOkFile.CheckExists()
         end
 
+    // This test stays in FsharpSuite for a later migration phases, it uses hardcoded #r to a C# compiled cslib.dll inside
```
Comment (psfinaki) [inline tests/fsharp/tests.fs:856]: Does this still apply then?

Comment (majocha) [reply tests/fsharp/tests.fs:856]: I have no idea, the comment was there before :)

Comment (psfinaki) [reply tests/fsharp/tests.fs:856]: No I know, but from what I understand - you are dehardcoding this cslib.dll thing hence the comment might not apply anymore :)

Comment (majocha) [reply tests/fsharp/tests.fs:856]: Yes and no, there in fact is a hardcoded #r inside.

Comment (psfinaki) [reply tests/fsharp/tests.fs:856]: I see. Well I guess this can go anyway, but can probably stay either, doesn't matter.

**Review by T-Gro — APPROVED** [review]

**Review by psfinaki — APPROVED** [review]
Thanks for dehardcoding this voodoo here, this helps a lot :)

**Pull request merged**